src/db.py

```diff
@@ -66,23 +74,21 @@ def reinit(self):
             session.query(Receipt).delete()
         except Exception as err:
             print(repr(err))
-
         try:
             Base.metadata.create_all(self._engine)
         except Exception as err:
             print(repr(err))
 
     def __del__(self):
         pass
-        #self._session.close()
+        # self._session.close()
 
     def add_user(self, user: dict):
         session = self._sm()
-
         c = session.query(User.id).filter(User.tg_id == user["tg_id"]).count()
         if c > 0:
+            _logger.warning("User already exists in database. STATUS_USER_ALREADY_EXIST")
```

Comment:
Need to add which user

_logger.warning("User %d already exists in database. STATUS_USER_ALREADY_EXIST", user["tg_id"])